react/tic-tac-toe/src/app/components/AuthenticatedRoutes/index.js

```diff
@@ -13,22 +12,14 @@ class AuthenticatedRoutes extends Component {
   ];
 
   render() {
-    return this.props.token ? this.routes.map(each => (
+    return window.localStorage.getItem('token') ? this.routes.map(each => (
```

Comment:
I wouldn't handle this here. Why don't you set this as default state in your store, specifically in your token key. You can check whether or not you have something in your localStorage and then set your default state